src/components/userDasboard/components/DashboardDetails.jsx

```diff
@@ -14,9 +14,6 @@ const DashboardDetails = ({ numberOfOrgs, numberOfDictionary, organizations }) =
             the traditional OCL
           </a>.
         </p>
-        <p className="lead">
-          You can <Link to="dashboard/dictionaries">view all OpenMRS public dictionaries</Link>.
```

Comment:
Are these deletions part of the ticket?

Reply:
This was not part of the task scope, I have removed it.